docs_src/design/adr/0026-Common Configuration.md

```diff
@@ -25,7 +28,15 @@ During bootstrapping, each service will either load the common configuration fro
 
 An additional common configuration setting must be present to indicate all other common settings have been pushed to the Configuration Provider. This setting is stored last and the services must wait for this setting to be present prior to pulling the common settings.
 
-### Configuration Bootstrapping
+Environment overrides are only applied when configuration is loaded from file. The overridden values are pushed into the Configuration Provider, when used.
+
+### Common Config Bootstrapping
+
+The following flow chart demonstrates the bootstrapping  (seeding) of the common configuration when using the Configuration Provider.
+

```

Comment:
Should "Set setting in CP..." and "Push common config to CP" in this diagram not be reversed?

Reply:
Great catch! Fixed.